Tests/TuistKitTests/Services/InstallServiceTests.swift

```diff
@@ -53,6 +53,8 @@ final class InstallServiceTests: TuistUnitTestCase {
         // Given
         let stubbedPath = try temporaryPath()
 
+        manifestFilesLocator.locatePackageManifestStub = stubbedPath.appending(components: "Tuist", "Package.swift")
```

Comment:
Have you considered taking the opportunity to use `@Mockable` with `ManifestsFileLocating`?